package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "description": "A chatbox back-end plugin for graasp",
   "main": "dist/service-api.js",
+  "types": "types/index.d.ts",
   "repository": "git@github.com:graasp/graasp-plugin-chatbox.git",
   "author": "Alexandre Chau <alexandre.chau@alumni.epfl.ch>",
```

Comment:
Usually what we do (since multiple people will work on this) is that we put Graasp Association (or React EPFL) as the author and then we add our names in the `contributors` array.  E.g. https://github.com/graasp/graasp-desktop/blob/master/package.json